helper.psm1

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+using namespace System.Runtime.InteropServices
+
+$DLL_NAME = "Microsoft.ManifestTool.dll"
+$MANIFESTOOLNAME = "ManifestTool"
+$MANIFESTOOL_DIRECTORY = Join-Path $PSScriptRoot $MANIFESTOOLNAME
+$MANIFEST_TOOL_PATH = "$MANIFESTOOL_DIRECTORY/$DLL_NAME"
+
+function Get-ManifestToolPath
+{
+    if (Test-Path $MANIFEST_TOOL_PATH)
+    {
+        return $MANIFEST_TOOL_PATH
+    }
+    throw "The SBOM Manifest Tool is not installed. Please run Install-SBOMUtil -SBOMUtilSASUrl <SASUrl>"
+}
+
+function Install-SBOMUtil
+{
+    param(
+        [string]
+        $SBOMUtilSASUrl
+    )
+    
+    if ([string]::IsNullOrEmpty($SBOMUtilSASUrl))
+    {
+        throw "The `$SBOMUtilSASUrl parameter cannot be null or empty when specifying `$(addSBOM)"
+    }
+
+    Write-Host "Installing $MANIFESTOOLNAME..."
+    Remove-Item -Recurse -Force $MANIFESTOOL_DIRECTORY -ErrorAction Ignore
+
+    Invoke-RestMethod -Uri $SBOMUtilSASUrl -OutFile "$MANIFESTOOL_DIRECTORY.zip"
+    Expand-Archive "$MANIFESTOOL_DIRECTORY.zip" -DestinationPath $MANIFESTOOL_DIRECTORY
+    
+    if (-not (Test-Path $MANIFEST_TOOL_PATH))
+    {
+        throw "$MANIFESTOOL_DIRECTORY does not contain '$DLL_NAME'"
+    }
+
+    Write-Host 'Done.'
+
+    return $MANIFEST_TOOL_PATH
+}
+
+$IsWindowsEnv = [RuntimeInformation]::IsOSPlatform([OSPlatform]::Windows)
```

Comment:
This type is not available in `pwsh`. Instead you can use `$IsWindows`, which is available in PowerShell 6 or higher. In addition, please add this `#Requires -Version 6.0` to the top of the script. For a full example, please see https://github.com/Azure/azure-functions-powershell-worker/blob/dev/build.ps1#L40.